test/run_test.py

```diff
@@ -533,6 +538,11 @@ def get_selected_tests(options):
         assert num_shards <= len(selected_tests), f"Number of shards must be less than {len(selected_tests)}"
         selected_tests = selected_tests[which_shard - 1 :: num_shards]
 
+    if options.exclude_jit_executor:
+        to_exclude = ['test_jit_cuda_fuser_profiling', 'test_jit_cuda_fuser_legacy', 'test_jit_profiling',
+                      'test_jit_legacy', 'test_jit_fuser_legacy', 'test_jit_fuser_te', 'test_tensorexpr']
+        options.exclude = list(set(options.exclude) | set(to_exclude))
```

Comment:
There is no harm in excluding the same test twice, is there?
Also, can you please move the list to one of the constants at the top of the file?
```suggestion
        options.exclude.extend( ['test_jit_cuda_fuser_profiling', 'test_jit_cuda_fuser_legacy', 'test_jit_profiling',
                      'test_jit_legacy', 'test_jit_fuser_legacy', 'test_jit_fuser_te', 'test_tensorexpr'])
```

Reply:
I think the one downside to having duplicates is that the print statement will run twice in the exclude_tests function